lib/workers/repository/updates/branchify.ts

```
@@ -72,9 +72,9 @@ export async function branchifyUpgrades(
       })
       .reverse();
 
-    if (config.fetchReleaseNotes && config.repoIsOnboarded) {
-      const branches = branchUpgrades[branchName].filter((upg) =>
-        needsChangelogs(upg)
+    if (config.fetchReleaseNotes !== 'off' && config.repoIsOnboarded) {
+      const branches = branchUpgrades[branchName].filter(
+        (upg) => needsChangelogs(upg) || config.fetchReleaseNotes === 'branch'
       );
       if (branches.length) {
```

same here, i would now throw an error of it's not set to branch.